packages/wdio-sync/webdriverio.d.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference types="./webdriverio-core"/>
+/// <reference types="wdio/sync/webdriverio-core"/>
```

Comment:
this one was good but not now :)

we should have @ here
@wdio/sync/webdriverio-core

Reply:
🤦‍♂️